newm-server/src/main/kotlin/io/newm/server/features/idenfy/model/IdenfyRequest.kt

```diff
@@ -0,0 +1,37 @@
+package io.newm.server.features.idenfy.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class IdenfyRequest(
+    @SerialName("clientId")
+    val clientId: String,
+    @SerialName("final")
+    val isFinal: Boolean,
+    @SerialName("status")
+    val status: Status
+) {
+    val isApproved: Boolean
```

Comment:
convert this to `by lazy { ... }` instead of `get`

Reply:
Done.